tools/cldr-code/src/main/java/org/unicode/cldr/util/Annotations.java

```diff
@@ -1,5 +1,19 @@
 package org.unicode.cldr.util;
 
+import com.google.common.base.Joiner;
```

Comment:
is it going to conflict with eclipse and intellij's import structure?

Reply:
maybe in the same sense that it will conflict with their java formatting in general -- anytime we edit a file, regardless of which IDE or editor we use, we'll need to re-run prettier -- though we might get the IDE/editor to run it automatically, in which case i don't know if the IDE/editor will get in a fight with itself over the imports